tests/refs/example_3.xyz

```diff
@@ -1,26 +1,26 @@
 24
-01/25/2018 14:25, XYZ structure generated by mol3D Class, molSimplify
+01/31/2022 21:43, XYZ structure generated by mol3D Class, molSimplify
 Fe 	0.164876	-0.191727	0.002225
-C 	-1.021696	-0.649640	1.769208
-C 	-0.750454	0.738902	1.740451
-C 	0.672141	0.863441	1.728410
-C 	1.259133	-0.442863	1.738427
-C 	0.192372	-1.396446	1.725351
-C 	-1.018398	-0.476177	-1.754685
-C 	-0.587096	0.877515	-1.699816
-C 	0.838182	0.886795	-1.673280
-C 	1.296601	-0.468747	-1.713600
-C 	0.137642	-1.312435	-1.758489
-H 	-1.495682	1.512203	1.546382
-H 	1.211675	1.800042	1.696211
-H 	2.315699	-0.671710	1.740902
-H 	0.272549	-2.474122	1.723548
-H 	-2.068461	-0.729567	-1.618995
-H 	-1.272056	1.690833	-1.462700
-H 	1.466732	1.763151	-1.587051
-H 	2.328095	-0.794209	-1.697841
-H 	0.143946	-2.394224	-1.769335
+C 	-0.301810	-0.044597	3.115961
+C 	-0.097802	1.195336	2.708077
+C 	0.854382	1.244102	1.700947
+C 	1.096277	-0.115829	1.188947
+C 	0.501112	-0.934865	2.274588
+C 	-1.272090	-0.401403	-2.505267
+C 	-0.860629	0.938543	-2.459898
+C 	0.382308	1.020861	-1.909600
+C 	0.739670	-0.270250	-1.374780
+C 	-0.277750	-1.176716	-1.956897
+H 	-0.646492	2.044971	3.072945
+H 	1.170320	2.143290	1.193163
+H 	2.160049	-0.390581	1.022001
+H 	0.404028	-2.016707	2.265259
+H 	-2.219304	-0.775337	-2.863406
+H 	-1.440798	1.785438	-2.811193
+H 	0.927166	1.934097	-1.723537
+H 	1.791302	-0.659853	-1.490443
+H 	-0.357312	-2.256132	-1.835639
 C 	-2.428227	-1.276701	1.761109
```

Comment:
Shows that only one carbon atom is frozen as "connecting  atom" for this complex. Not sure if this is intended behavior.